src/whylogs/app/session.py

```diff
@@ -38,7 +38,8 @@ def __init__(
         writers: List[Writer],
         verbose: bool = False,
         with_rotation_time: str = None,
-        cache_size: int = None
+        cache_size: int = None,
+        useSongbird: bool = False,
```

Comment:
shouldnt you check the writer list for whylabs writer ?

Reply:
or this parameter add a whylabs writer

Reply:
Yeah it's duplicated with `writers` up there. I don't think the parameter should be there